packages/react-instantsearch-hooks-web/src/widgets/__tests__/Stats.test.tsx

```diff
@@ -168,6 +168,45 @@ describe('Stats', () => {
     });
   });
 
+  test('renders with deprecated translations (with a deprecation warning)', async () => {
+    const client = createMockedSearchClient();
+    let result: ReturnType<typeof render> | undefined = undefined;
+
+    expect(async () => {
```

Comment:
```suggestion
    await expect(async () => {
```


I think this is what the lint warning is for?